### ISSUE TITLE
Performance issues

### ISSUE THREAD

**Issue by SPAstef** [animus-coop/vscode-phpstorm-formatter#4]
Hi, I don't know if this is intended/inevitable, but formatting with this plugin makes my PC fans turn on and takes ~10 seconds time. I love the simplicity of customization of PHPstorm formatter, and that it covers all web-related languages (no more 10 different formatters, yay!), but is there any way to make this run faster and without overheating the CPU?

**Comment by manuvalle** [comment]
Hello @SPAstef Yes, at the moment the formatting process is pretty slow but I don't think there is something we can do about it, what makes it slow is the formatting script provided by PHPStorm. Each time you format something the PHPStorm formatting script is executed and launches IntelliJ IDEA, which takes a while to start up. It's like opening PHPStorm, formatting a file and closing it after it.

https://www.jetbrains.com/help/idea/command-line-formatter.html

**Comment by SPAstef** [comment]
Yeah I guessed so... Maybe it is possible to launch it and leave it in background, then passing it the file to format waking up the process and then pausing it again?
